src/commands/Management/createMute.ts

```diff
@@ -26,6 +28,7 @@ export default class extends SkyraCommand {
 			await message.guild!.settings.update(GuildSettings.Roles.Muted, role, {
 				extraContext: { author: message.author.id }
 			});
+			await message.react(resolveEmoji(Emojis.GreenTick)!);
```

Comment:
The permissions are not checked, you're taking for granted that Skyra will have all `ADD_REACTIONS`, `USE_EXTERNAL_EMOJIS`, and `READ_CHANNEL_HISTORY` permissions, while it's rare for her to lack any of those, it can happen.

Reply:
I've added the extra permissions to `requiredGuildPermissions`.